### PR TITLE
{bio}[foss/2019a,fosscuda/2019a] Xmipp v3.19.02 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Depends on easyblock https://github.com/easybuilders/easybuild-easyblocks/pull/1837
Depends on ~~#9190~~, #9191

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/0008e601c834fa26d6447f27dc8791d7 for a full test report.

File: easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippCore-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/I2PC/xmippViz/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippViz-v%(version)s.tar.gz',
+    },
+    {
+        'git_config': {
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb]: using `git_config` causes trouble for checksums (see https://github.com/easybuilders/easybuild-framework/issues/2726)

But you don't need this at all when downloading from GitHub, see https://github.com/DStrelak/cuFFTAdvisor/archive/14523b8ca7e2f5474feae49eb5ac8ca1fa25f5cd.tar.gz.

So:

```python
{
    'source_urls': ['https://github.com/DStrelak/cuFFTAdvisor/archive/'],
    'download_filename': '14523b8ca7e2f5474feae49eb5ac8ca1fa25f5cd.tar.gz',
    'filename': 'cuFFTAdvisor-20191010.tar.gz',
}

File: easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+models of macromolecular complexes using Electron Microscopy (3DEM). It
+integrates several software packages and presents an unified interface
+for both biologists and developers. Scipion allows to execute workflows
+combining different software tools, while taking care of formats and
+conversions. Additionally, all steps are tracked and can be reproduced
+later on. 
+"""
+
+toolchain = {'name': 'fosscuda', 'version': '2019a'}
+# toolchainopts = {'openmp': True, 'noopt': True, 'debug': True }
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb]: should be removed?

File: easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02_correct_order_of_cu_includes.patch [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cuda_gpu_multidim_array.cu is just an include file containing templates.
+Needs to be before cuda_gpu_geo_transformer.cu
```
Comment (boegel) [inline easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02_correct_order_of_cu_includes.patch]: How did this ever work?!

File: easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb]: oops, didn't think of that...

File: easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb (hunk 2 above)
Comment (akesandgren) [reply easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb]: yeah, forgot it.

File: easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02_correct_order_of_cu_includes.patch (hunk 3 above)
Comment (akesandgren) [reply easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02_correct_order_of_cu_includes.patch]: don't know....

**Comment by akesandgren** [comment]
Anyway, 3.19.02 is too old for Scipion 2.0.0. So I'll close this one and upload a newer version.

**Comment by akesandgren** [comment]
Replaced by https://github.com/easybuilders/easybuild-easyconfigs/pull/9257

**Pull request closed without merging**